test/system/irohad_test.cpp

```diff
@@ -67,11 +67,20 @@ class IrohadTest : public AcceptanceFixture {
   }
 
   void launchIroha(const std::string &parameters) {
-    iroha_process_.emplace(irohad_executable.string() + parameters);
-    std::this_thread::sleep_for(kTimeout);
+    boost::process::ipstream output;
+    std::thread wait_init([&output] {
+        std::string line;
+        while (std::getline(output, line)) {
+          if (line.find("iroha initialized") != std::string::npos) {
```

Comment:
There should be some another way to fix that. For now it is okay I think. But please leave todo that will require to remove this check as soon as healthcare is introduced

Reply:
@kamilsa what can be a healthcare in this case?